src/msw/headerctrl.cpp

```diff
@@ -123,6 +123,9 @@ class wxMSWHeaderCtrl : public wxControl
                            int sizeFlags = wxSIZE_AUTO) wxOVERRIDE;
     virtual void MSWUpdateFontOnDPIChange(const wxSize& newDPI) wxOVERRIDE;
 
+    // This function can be used as event handle for wxEVT_DPI_CHANGED event.
+    void WXHandleDPIChanged(wxDPIChangedEvent& event);
```

Comment:
Is it possible to name these functions `OnDPIChanged`? That is what we used before with `wxDPIChangedEvent`, and it will have the same `On***` format as lots of other event handler functions.

I see you used it in some other places as well, so those might have to be changed as well.

But wait for @vadz opinion on this before doing this.

Reply:
Actually calling this function like this was my idea. The rationale is that using this name should avoid any potential clashes with `OnXXX()` functions defined in the user code. I agree that this name is unusual and looks kind of ugly, but this is the point -- hopefully nobody would be tempted to neither define such function in their code nor even to call the existing wx function with such weird name directly.

Do you think it's too ugly/inconsistent with the other handlers to be worth it?

Reply:
I don't see why this handler is any different from other handlers. Most classes have `OnXXX()` functions that a user should never call.

I'm indifferent to keeping it or changing it. It's just something I noticed while checking the PRs.

Reply:
I know that many other classes use `OnXXX()` and I don't think it's a problem big enough to justify changing them, but I still think it would be better to avoid possible clashes with user-defined functions, that often use the same naming convention, so I try to avoid introducing any more of them.

IOW I'd like to keep this naming here, if nobody else objects to it strongly.